azext_iot/operations/hub.py

```diff
@@ -9,25 +9,27 @@
 import six
 from knack.log import get_logger
 from knack.util import CLIError
-from azext_iot.constants import (EXTENSION_ROOT,
-                                 DEVICE_DEVICESCOPE_PREFIX,
-                                 TRACING_PROPERTY,
-                                 TRACING_ALLOWED_FOR_LOCATION,
-                                 TRACING_ALLOWED_FOR_SKU)
+from azext_iot.constants import (
+    EXTENSION_ROOT,
+    DEVICE_DEVICESCOPE_PREFIX,
+    TRACING_PROPERTY,
+    TRACING_ALLOWED_FOR_LOCATION,
+    TRACING_ALLOWED_FOR_SKU,
+)
 from azext_iot.common.sas_token_auth import SasTokenAuthentication
-from azext_iot.common.shared import (DeviceAuthType,
-                                     SdkType,
-                                     ProtocolType,
-                                     ConfigType)
+from azext_iot.common.shared import DeviceAuthType, SdkType, ProtocolType, ConfigType
 from azext_iot.common._azure import get_iot_hub_connection_string
-from azext_iot.common.utility import (shell_safe_json_parse,
-                                      read_file_content,
-                                      validate_key_value_pairs,
-                                      url_encode_dict,
-                                      unpack_msrest_error,
-                                      init_monitoring,
-                                      process_json_arg)
+from azext_iot.common.utility import (
+    shell_safe_json_parse,
+    read_file_content,
+    validate_key_value_pairs,
+    url_encode_dict,
+    unpack_msrest_error,
+    init_monitoring,
+    process_json_arg,
+)
 from azext_iot._factory import SdkResolver, CloudError
+from azext_iot.operations import events3
```

Comment:
There are way too many formatting changes in this module to difference between actual logic changes.

Can you save me time and itemize the core changes along with this one?

Also, while you are here, you could clean-up usage of `importlib` that's a legacy remnant from supporting Python 2.7 and preventing conflicts from static checkers. Should be replaced with natural import syntax since we are 3.5+ now.

Reply:
Sounds like we're on the same page, the only change I was making here was removing `importlib`.

Went ahead and did it for the rest of the file too